UnitySDK/Assets/ML-Agents/Scripts/InferenceBrain/BarracudaModelParamLoader.cs

```diff
@@ -138,46 +156,71 @@ public static IEnumerable<string> CheckModel(Model model, BrainParameters brainP
                 return failedModelChecks;
             }
 
-            var modelApiVersion = (int)model.GetTensorByName(TensorNames.VersionNumber)[0];
-            var memorySize = (int)model.GetTensorByName(TensorNames.MemorySize)[0];
-            var isContinuousInt = (int)model.GetTensorByName(TensorNames.IsContinuousControl)[0];
-            var isContinuous = GetActionType(isContinuousInt);
-            var actionSize = (int)model.GetTensorByName(TensorNames.ActionOutputShape)[0];
-            if (modelApiVersion == -1)
+            // DEBUG
+            var layerNames = model.GetLayerNames();
+            var outString = "";
+            foreach(var layerName in layerNames)
             {
-                failedModelChecks.Add(
-                    "Model was not trained using the right version of ML-Agents. " +
-                    "Cannot use this model.");
-                return failedModelChecks;
+                outString += $"{layerName}, ";
             }
-            if (modelApiVersion != k_ApiVersion)
+            // END DEBUG
+
+            try
             {
-                failedModelChecks.Add(
-                    $"Version of the trainer the model was trained with ({modelApiVersion}) " +
-                    $"is not compatible with the Brain's version ({k_ApiVersion}).");
-                return failedModelChecks;
-            }
 
-            failedModelChecks.AddRange(
-                CheckIntScalarPresenceHelper(new Dictionary<string, int>()
+//                var modelApiVersion = (int)model.GetTensorByName(TensorNames.VersionNumber)[0];
+//                var memorySize = (int)model.GetTensorByName(TensorNames.MemorySize)[0];
+//                var isContinuousInt = (int)model.GetTensorByName(TensorNames.IsContinuousControl)[0];
+//                var isContinuous = GetActionType(isContinuousInt);
+//                var actionSize = (int)model.GetTensorByName(TensorNames.ActionOutputShape)[0];
+                var modelApiVersion = 2;
```

Comment:
Hacked this in for now to see what else broke in the importing. Currently the desired magic strings (i.e. TensorNames.VectorObservationPlacholder) are also not found (because the model has them suffixed with `:0`